resources/resources/tests/test-aws-compute-ebs-encrypted-volumes.yaml

```diff
@@ -0,0 +1,93 @@
+ruleId: aws-compute-ebs-encrypted-volumes
```

Comment:
We will need to revise the naming of these new rules to match the latest 'schema' for magpie rules. https://openraven.atlassian.net/wiki/spaces/~622d5e861c09d2007012857e/pages/1680080922/Integration+of+Magpie+Policies+and+Rules

e.g

id: 71ddc520-83f3-499e-9a71-b8e7d4aacc3a
# opnrvn-r-149
refId: aws-application-services-sns-encrypted-kms
type: asset
name: >
  Ensure Amazon SNS Topic is Encrypted with AWS KMS Key
description: >
  Ensure Amazon SNS Topic is Encrypted with AWS KMS Key.
severity: high
enabled: true
sql: >
  SELECT arn as assetid
remediationDocURLs: https://github.com/openraven/security-rules/wiki
version: 1.0